src/primitives.rs

```diff
@@ -4,8 +4,8 @@ pub use ssz_rs::prelude::Root;
 
 pub type Slot = u64;
 pub type Epoch = u64;
-pub type CommitteeIndex = u64;
-pub type ValidatorIndex = u64;
+pub type CommitteeIndex = usize;
+pub type ValidatorIndex = usize;
```

Comment:
i think this makes sense.

hopefully we won't need to do anything silly to serialize as a `u64` one day